tests/bson/bson-utcdatetime-005.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+BSON BSON\UTCDateTime construction from DateTime
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$tests = [
+    new DateTime(),
+    new DateTime('@1215282385'),
+    new DateTime('2011-01-01T15:03:01.012345Z'),
+    new DateTime('2050-11-12T13:14:15.999999Z'),
+];
+
+foreach ($tests as $test) {
+    var_dump(new MongoDB\BSON\UTCDateTime($test));
```

Comment:
Sounds like a good idea. Let's get https://github.com/mongodb/mongo-php-driver/pull/338 merged first in that case, since it fixes `toDateTime()` and the expected output.